src/main/java/org/spongepowered/api/data/key/Keys.java

```diff
@@ -708,6 +714,12 @@
 
     public static final Key<Value<Vector3d>> RIGHT_ARM_ROTATION = null;
 
+    /**
+     * Represents the {@link Key} for representing whether something is right
+     * handed.
+     */
+    public static final Key<Value<Boolean>> RIGHT_HANDED = null;
```

Comment:
pseudo enum? I imagine it's very unlikely that a 3rd handedness will be invented in the future haha